src/constructs/kinesis/kinesis-stream.ts

```diff
@@ -22,7 +21,7 @@ export interface GuKinesisStreamProps extends StreamProps, GuMigratingResource {
  * new GuKinesisStream(stack, "LoggingStream", { existingLogicalId: "MyCloudFormedKinesisStream" });
  * ```
  */
-export class GuKinesisStream extends GuStatefulMigratableConstruct(Stream) {
+export class GuKinesisStream extends Stream {
```

Comment:
This construct is now a pure alias to the AWS one and isn't needed. I'd suggest we remove it, but not sure if it's correct to do here or in it's own PR?